CHANGES.rst

```diff
@@ -491,6 +491,15 @@ astropy.visualization
 astropy.wcs
 ^^^^^^^^^^^
 
+- Add .upper() to ctype or ctype names to wcsapi/fitwcs.py to mitigate bugs from
+  unintended lower/upper case issues [#10557]
+- Added bounds to ``fit_wcs_from_points`` to ensure CRPIX is on
+  input image. [#10346]
+
```

Comment:
@nden This somehow got left from the merge commit or maybe in some other way. The resulting file will have two entries for the same bug fix in `wcsapi`.

```suggestion
```